### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by lemuelf**
Creates the SDK.

### CODE REVIEW

**Comment by arnoldchoa91** [comment]
This is from README.md.
```
bosta = Bosta(
    api_key=os.getenv('BOSTA_API_KEY'), api_base=os.getenv('BOSTA_API_KEY'))
```
I think you meant to put the API url for `api_base`.

**Comment by lemuelf** [comment]
@arnoldchoa91 @kinoki @mickorivera Can you guys please review or suggest improvements for this?  Thanks.

**Comment by kinoki** [comment]
Tested CRUD, working fine. Please update readme as type is now required and correct dropOffAddress.

File: bosta/base.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def airwaybill_get(self,
         arg = airwaybill.GetAirWaybillArg(_id)
         r = self.request(
             airwaybill.get,
-            'airwaybill',
+            'deliveries/awb',
```
Comment (lemuelf) [inline bosta/base.py]: Was this change auto-generated?

Comment (lemuelf) [reply bosta/base.py]: Doesn't look like it.  I guess it's either we change the namespace to produce this desired result, or we rewrite the client to use a different attribute to build the endpoint URL.

**Pull request merged**